backend/tests/api/test_manage_user.py

```diff
@@ -0,0 +1,90 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+from rest_framework.test import APIClient
+from rest_framework import status
+
+ME_URL = reverse('meal:me')
+
+
+def create_user(**kwargs):
+    return get_user_model().objects.create_user(**kwargs)
+
+
+class PublicUserApiTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.testTables = [
+            {
+                'name': 'Test that unauthorized user cannot retrieve profile',
+                'request': (lambda: self.client.get(ME_URL)),
+                'expected_status_code': status.HTTP_401_UNAUTHORIZED,
+            },
+            {
+                'name': 'Test that unauthorized user cannot POST profile',
```

Comment:
isn't it exactly the same test as per line 31? same endpoint, same expected status, just different name. Please remove one.

Reply:
OK